raw1394util.c

```diff
@@ -150,12 +152,14 @@ int discoverAVC( int* port, octlet_t* guid )
 				}
 				if ( ( ( rom1394_get_node_type( &rom_dir ) == ROM1394_NODE_TYPE_AVC ) &&
 				         avc1394_check_subunit_type( handle, i, AVC1394_SUBUNIT_TYPE_VCR ) ) ||
-				       ( rom_dir.unit_spec_id == MOTDCT_SPEC_ID ) )
+				       ( rom_dir.unit_spec_id == MOTDCT_SPEC_ID ) ||
+				       ( rom_dir.vendor_id == DVMCDA1_VENDOR_ID &&
+				         rom_dir.model_id == DVMCDA1_MODEL_ID ) )
 				{
 					rom1394_free_directory( &rom_dir );
 					octlet_t my_guid, *pguid = ( *guid == 1 )? guid : &my_guid;
 					*pguid = rom1394_get_guid( handle, i );
-					fprintf( stderr, "Found AV/C device with GUID 0x%08x%08x\n",
+					fprintf( stderr, "Found video device with GUID 0x%08x%08x\n",
```

Comment:
Do not change this word. There are programs that run `dvgrab` as a child program and parse its output. In that case, stdin and stderr are like a protocol that should only be changed if necessary.

Reply:
I did Google searches for multiple substrings from this message and have found no evidence of any code that is looking for this particular message. Nonetheless, I have amended the commit and undone the change to the message, as per your request.